### PR TITLE
Firsttask

### PR DESCRIPTION
**Pull request by antonovaea**
-get request hundler 
-unic id and unic token were created by uuidv4 function
-map was created with uuid as key and token as value

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -2,18 +2,30 @@ const express = require('express')
...
-
+let map = new map(); //creating map for uuid as key and token as value
 server.use('*', cors()) 
 
   
 /**
  *  this endpoint opens session
  */
 server.get('/create', (req, res) => {
+    const { v4: uuidv4 } = require('uuid');
```
Comment (sumbarbarrian) [inline server/server.js]: Please move this call to the top. To 4th line. This is method called for each '/create' request. It means that `require` method wil also be called each request.

File: server/server.js [hunk 2]
```diff
@@ -2,18 +2,30 @@ const express = require('express')
 const ws = require('ws')
 const cors = require('cors')
 const server = express()
-
+let map = new map(); //creating map for uuid as key and token as value
```
Comment (sumbarbarrian) [inline server/server.js]: i'm afraid that here will be an error. Because JS has `Map` class but hasn't  `map`

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (sumbarbarrian) [inline package-lock.json]: please remove this file

File: ui/package.json [hunk 4]
```diff
@@ -14,6 +14,7 @@
     "react-dom": "^17.0.2",
```
Comment (sumbarbarrian) [inline ui/package.json]: this package is not used in UI project. Please cleanup changes from UI.

File: ui/package-lock.json [hunk 5]
```diff
@@ -14892,8 +14892,7 @@
     "uuid": {
       "version": "8.3.2",
       "resolved": "https://registry.npmjs.org/uuid/-/uuid-8.3.2.tgz",
-      "integrity": "sha512-+NYs2QeMWy+GWFOEm9xnn6HCDp0l7QBD7ml8zLUmJ+93Q5NF0NocErnwkTkXVFNiX3/fpC6afS8Dhb/gz7R7eg==",
-      "optional": true
+      "integrity": "sha512-+NYs2QeMWy+GWFOEm9xnn6HCDp0l7QBD7ml8zLUmJ+93Q5NF0NocErnwkTkXVFNiX3/fpC6afS8Dhb/gz7R7eg=="
```
Comment (sumbarbarrian) [inline ui/package-lock.json]: get rid of this please :)

Comment (antonovaea) [reply ui/package-lock.json]: package-lock.json deleted at all

File: ui/package.json (hunk 4 above)
Comment (antonovaea) [reply ui/package.json]: done :)

File: server/server.js (hunk 2 above)
Comment (antonovaea) [reply server/server.js]: it was fixed

File: package-lock.json (hunk 3 above)
Comment (antonovaea) [reply package-lock.json]: done

File: server/server.js (hunk 1 above)
Comment (antonovaea) [reply server/server.js]: replaced

**Review by sumbarbarrian — APPROVED** [review]

**Pull request merged**